apps/css-workshop/src/pages/checkbox.astro

```diff
@@ -1,5 +1,6 @@
 ---
 import Layout from './_layout.astro';
+import Checkbox_ from '../components/Checkbox.astro';
```

Comment:
I believe there are checkboxes in other demo files too. E.g. `information-panel.astro`, `table.astro`, etc. Those would need to use the `Checkbox_` component too.

Reply:
I'll switch those over!